server/routes/api/landing_page.py

```diff
@@ -518,20 +556,22 @@ def populate_category_data(category):
             # If there is no data for a category in overview page, need to
             # "borrow" it from the category page.
             if not has_data(data):
-                populate_category_data(category)
-                total_charts = 0
-                spec_and_stat[OVERVIEW][category] = []
-                for topic in spec_and_stat[category]:
-                    spec_and_stat[OVERVIEW][category].extend(
-                        spec_and_stat[category][topic])
-                    total_charts += len(spec_and_stat[category][topic])
-                    if total_charts > MAX_OVERVIEW_CHART_GROUP:
-                        break
+                populate_additional_category_data(category)
 
     # Get chart category name translations
-    categories = {}
-    for category in list(spec_and_stat.keys()) + list(spec_and_stat[OVERVIEW]):
-        categories[category] = gettext(f'CHART_TITLE-CHART_CATEGORY-{category}')
+    ordered_category_dict = {}
+    all_categories = set(
+        list(spec_and_stat.keys()) + list(spec_and_stat[OVERVIEW]) +
+        raw_page_data["validCategories"][dcid]['category'])
```

Comment:
sorry i haven't been following this change too closely -- but what is `validCategories`?

Reply:
It holds the categories that has data for a place. It's used to determine what categories to show on the LHS menu

Reply:
ok sg, thanks!